recipes-multimedia/libcamera-apps/rpi-libcamera-apps_git.bb

```diff
@@ -30,7 +30,8 @@ EXTRA_OECMAKE = "\
 
 LIBCAMERA_ARCH = "${TARGET_ARCH}"
 LIBCAMERA_ARCH:aarch64 = "arm64"
-LIBCAMERA_ARCH:arm = "armv8-neon"
+LIBCAMERA_ARCH:raspberrypi3 = "armv8-neon"
+LIBCAMERA_ARCH:raspberrypi4 = "armv8-neon"
```

Comment:
Use both overrides:
LIBCAMERA_ARCH:arm:raspberrypi3 = "armv8-neon"
LIBCAMERA_ARCH:arm:raspberrypi4 = "armv8-neon"

to avoid setting "armv8-neon" for raspberrypi3-64 and raspberrypi4-64

Reply:
that now reminds me, why it was overridden in first place for `arm` override. Actually it was for 32bit systems.

Reply:
Indeed armv8-neon should only be applied for 32bit systems. Fixed, thanks!

Reply:
Looking at later removed CMake files:

https://github.com/raspberrypi/libcamera-apps/commit/9e172655e257972e5da45d59a720192516f7d769#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aL27

for raspberrpi2 it will set TARGET_ARCH which won't be recognized by CMake, but that just won't add -ftree-vectorize for rpi2, which is probably fine.